plugins/att_cybersecurity_alienvault_otx/komand_att_cybersecurity_alienvault_otx/util/utils.py

```diff
@@ -9,7 +9,10 @@
 }
 
 
-def get_indicatortypes(indicator_type):
-    if indicator_type not in indicator_types:
-        raise Exception(f"Indicator type: {indicator_type} is not a supported indicator type")
-    return indicator_types.get(indicator_type)
+def get_indicator_type(indicator_type):
+    if indicator_type in indicator_types:
+        return indicator_types.get(indicator_type)
+    raise PluginException(
+        cause=f"Indicator type {indicator_type} is not a supported indicator type.",
+        assistance="Please provide a valid indicator type.",
+    )
```

Comment:
Do you use that messages in some other place? I always try to put all the messages into separate variables, but it's up to you

Reply:
No, this is the only place I use it, so I think at the moment there is no need to create separate variables for these messages :)